src/plots/barPlot.ts

```diff
@@ -291,6 +291,7 @@ export module Plots {
     }
 
     protected _visibleOnPlot(datum: any, pixelPoint: Point, selection: d3.Selection<void>): boolean {
+      Utils.Window.deprecated("v1.1.0");
```

Comment:
Change all the deprecated functions to aim at v1.2.0?

As a side note, it'd be nice if we can find a way to automate this deprecated idea.